core/cmd/local_client_p2p.go

```diff
@@ -52,3 +66,63 @@ func (cli *Client) createP2PKey(c *clipkg.Context) error {
 	fmt.Printf(createKeyMsg, enc.ID, enc.PubKey, peerID.Pretty())
 	return nil
 }
+
+func (cli *Client) listP2PKeys(c *clipkg.Context) error {
+	cli.Config.Dialect = orm.DialectPostgresWithoutLock
+	store := cli.AppFactory.NewApplication(cli.Config).GetStore()
+	keys, err := store.FindEncryptedP2PKeys()
+	if err != nil {
+		return errors.Wrapf(err, "while fetching encrypted OCR key bundles")
+	}
+
+	fmt.Println(
+		`***********************************************************************************
+Encrypted P2P keys in DB
+***********************************************************************************`)
+	for keyidx, key := range keys {
+		fmt.Println("ID                ", key.ID)
+		fmt.Println("PeerID            ", key.PeerID)
```

Comment:
What type is this? If its a `peer.ID` from libp2p you'll need to call `.Pretty()` on it or you get something surprising

Reply:
looks like the string method is an alias for Pretty

Reply:
```

Reply:
Yup the confusing thing is that `peer.ID` itself is defined like this `type ID string` and the underlying string representation is NOT what you want. I don't know why they did it like this, seems mad to be but that's what it is.

So it's best to be explicit about calling either `String()` or `Pretty()` everyt ime.

Reply:
JK - turns out this is already a string of `PeerID.Pretty()` - because it's the PeerID on the `EncryptedP2PKey` struct. So we're covered here :)